app/templates/Articles/add.php

```diff
@@ -0,0 +1,12 @@
+<!-- File: templates/Articles/add.php -->
+
+<h1>Add Article</h1>
+<?php
+    echo $this->Form->create($article);
+    // Hard code the user for now.
```

Comment:
Understand we'll fix this later. What's the efficient way we'll get attention that the next line require fixing later?

Reply:
yes, I know, we need to take the user id from the current user logged

Reply:
> we need to take the user id from the current user logged

Understood. Back to my original question:

> What's the efficient way we'll get attention that the next line require fixing later?

Adding the TODO / FIXME can get more attention, another member can notice TODO / FIXME easily instead of a normal comment.

Reply:
Usually, we use 'TODO:', but I'm sorry, we can establish those conventions once we start the project

Reply:
Please check https://github.com/combimauri/tutorial-cakephp/pull/2#discussion_r1386720815

Reply:
I think having TO DO is ok or do you want me to change like it was in the beginning, I can't get it